logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationSeenDateUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.logic.feature.conversation
+
+import com.wire.kalium.logic.data.conversation.ConversationRepository
+import com.wire.kalium.logic.data.id.QualifiedID
+
+class UpdateConversationSeenDateUseCase(private val conversationRepository: ConversationRepository) {
+
+    suspend operator fun invoke(conversationId: QualifiedID, timeStamp: String) {
```

Comment:
```suggestion
    suspend operator fun invoke(conversationId: QualifiedID, timestamp: String) {
```

Why do we use a `String` and not something like `Instant`?

Reply:
I think I followed some convention from other use case but I agree Instant would be better here we use everywhere Clock.System.now() which returns an Instant, but can we relay on iOS and JS being able to provide us a Instant object ?